core/src/main/java/org/apache/cxf/common/jaxb/JAXBUtils.java

```diff
@@ -1174,6 +1176,54 @@ private static Object createEscapeHandler(Class<?> cls, String simpleClassName)
         }
         return null;
     }
-    
-
+    public static JAXBContext createJAXBContext(Set<Class<?>> classes,
```

Comment:
@jimma sorry what is the diff between createJAXBContext and createContext ? we probably need only one, right?

Reply:
That is duplicate. I removed one. Thanks for review @reta  .